runtime/dart_init.cc

```diff
@@ -261,8 +261,11 @@ Dart_Isolate IsolateCreateCallback(const char* script_uri,
   std::vector<uint8_t> snapshot_data;
   std::string entry_path;
   if (!IsRunningPrecompiledCode()) {
-    // Assert that entry script URI starts with file://
-    FTL_CHECK(entry_uri.find(kFileUriPrefix) == 0u);
+    // Check that the entry script URI starts with file://
+    if (entry_uri.find(kFileUriPrefix) != 0u) {
+      *error = strdup("Isolates must use file:// URIs");
```

Comment:
TIL error messages are malloc'ed and caller freed. BTW, there are other spots in this file where the error is logged but there is an FTL_CHECK right after. Maybe we want to return `nullptr` there as well?